examples/recognition_pipeline/pipeline_lib/redefine_labels.py

```diff
@@ -0,0 +1,43 @@
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+    https://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+"""
+
+import apache_beam as beam
+
+# TODO: Think about whether to do this in the same map, or change to ids inside the redefine method.
+LABEL_TO_ID = {'bag': 'GaveDiWni9CzOp6eYRDE'}
+REDEFINE_LABELS = {'Google_Vision_API':{'Bag': LABEL_TO_ID['bag'], 'Glass': LABEL_TO_ID['bag']}} #TODO: fill this map, need to think how do to that. 
```

Comment:
I think we agreed to have this mapping in a table on firestore, no?
Can you update the code to read it from there?

Reply:
To avoid having to manually fill the map: How about the following logic:
If there is no mapping specified: Automatically create a new label with the same name and map the label from the API to that (identity mapping).
As an example, if API `ellipsis` has label `bag` and there is no entry in the mapping table for that, create a new label `bag` (we can have multiple labels with name `bag`, thanks to using a generated ID) and map the ellipsis bag to that label. 
Then, for repeated labels, one can always manually change the mapping from `ellipsis bag` to an existing `bag` label or to some other label

Reply:
moved the map to the firestore databasem and changed the mapping to what we talked about during the meet.